chains/nomad-ethereum/Cargo.toml

```diff
@@ -11,6 +11,8 @@ edition = "2021"
 serde = "1.0.120"
 serde_json = { version = "1.0.61", default-features = false }
 ethers = {git = "https://github.com/gakonst/ethers-rs", branch = "master", features = ["abigen"]}
+ethers-core = {git = "https://github.com/gakonst/ethers-rs", branch = "master"}
+ethers-providers = {git = "https://github.com/gakonst/ethers-rs", branch = "master"}
```

Comment:
open an issue for simplifiying our ethers dependencies 😅

Reply:
https://github.com/nomad-xyz/rust/issues/42